taskcluster/fenix_taskgraph/manifests/fenix_candidates.yml

```diff
@@ -66,22 +66,22 @@ mapping:
         description: "Android package for arm64-v8a"
         pretty_name: fenix-${version}.${locale}.android-arm64-v8a.apk
         destinations:
-            - ${dated_path}-fenix-${version}-android-arm64-v8a
+            - ${folder_prefix}fenix-${version}-android-arm64-v8a
     armeabi-v7a/target.apk:
         <<: *default
         description: "Android package for armeabi-v7a"
         pretty_name: fenix-${version}.${locale}.android-armeabi-v7a.apk
         destinations:
-            - ${dated_path}-fenix-${version}-android-armeabi-v7a
+            - ${folder_prefix}fenix-${version}-android-armeabi-v7a
     x86/target.apk:
         <<: *default
         description: "Android package for x86"
         pretty_name: fenix-${version}.${locale}.android-x86.apk
         destinations:
-            - ${dated_path}-fenix-${version}-android-x86
+            - ${folder_prefix}fenix-${version}-android-x86
```

Comment:
We should probably either make it so `version.txt` is correct on nightly, or change the ${version} subst to use something else, so nightly filenames don't say "beta"?

Reply:
I'm guessing the fix may involve manually? automatically? bumping `main` to an alpha version, and bumping it to a beta version when we branch to go to beta. If these are off the same branch, then we may need another solution.

Reply:
nightly is off the `main` branch, beta and release are from a `releases_vXX.0` branch, as far as I know.

Reply:
Should I just create an issue for it?